### ISSUE TITLE
Have you modified the testing script?

### ISSUE THREAD

**Issue by Kobe972** [wine99/cs144-20fa#2]
In my version of code, _sender.ack_received is a function of type bool instead of void.

**Comment by wine99** [comment]
No, I think not. You can check using git blame. If they have changed some types in the template, it must be that they believe the implementation would become easier.
